nepalingo-web/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { useLocation } from "react-router-dom";
+
+const Home: React.FC = () => {
+  const location = useLocation();
+  const username = location.state?.username;
```

Comment:
This is a bit of an antipattern, we should use a context and pass it in through a provider instead. I'll add a followup ticket to refactor it here: https://github.com/nepalcodes/nepalingo/issues/60

Reply:
so that means I don't have to do it now, right? cool!